src/promptflow/tests/sdk_cli_test/e2etests/test_flow_test.py

```diff
@@ -127,7 +127,7 @@ def test_pf_test_with_additional_includes(self, caplog):
         assert all([key in FLOW_RESULT_KEYS for key in result])
 
         # Test additional includes don't exist
-        with pytest.raises(ValueError) as e:
+        with pytest.raises(UserErrorException) as e:
```

Comment:
this test will be skipped in any condition, so maybe no need to change this.

Reply:
this comment is optional, maybe take a look on below first

Reply:
Good catch, this error was thrown from SDK V2, it is indeed ValueError.

Reply:
I will modify it first to ensure that the code is correct.